src/snapshot/tosvg.js

```diff
@@ -114,7 +114,7 @@ module.exports = function toSVG(gd, format) {
             // Is this an IE thing? Any other attributes or style elements that can have quotes in them?
             // TODO: this looks like a noop right now - what happened to it?
             var ff = txt.style('font-family');
-            if(ff && ff.indexOf('"') !== -1) txt.style('font-family', ff.replace(/"/g, '"'));
+            if(ff && ff.indexOf('"') !== -1) txt.style('font-family', ff.replace(/"/g, '\\\''));
```

Comment:
I'm thinking this was needed for when we passed svg to the back end for image gen... Then we turned it to a noop when we started doing this same fix on the back end... Then it became moot when we started passing json.